packages/react-composites/src/composites/CallComposite/pages/CallPage.tsx

```diff
@@ -186,7 +230,7 @@ export const CallPage = (props: CallPageProps): JSX.Element => {
               <NetworkReconnectTile {...networkReconnectTileProps} isMobile={mobileView} />
             )
           ) : (
-            <></>
+            galleryContentWhenNotInCall
```

Comment:
Can you explain what is going on here real quick? 
I can see that you have a video tile and some persona data that has a bit of styling done. Could you attach a screenshot here and explain when this will hit?

Looks like it's supposed to be when you're not in a call? So is this inbetween states of going from one room to another room?

Reply:
The update here is to be able to show the call page when the breakout room call ends while the local user is momentarily not in a call and joining the main meeting. It will show VideoTile in the VideoGallery like shown in this video
https://github.com/user-attachments/assets/8263833a-14c7-4f22-a2eb-25cb6066be07